packages/uix-host/src/metrics.ts

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/**
+ * Adapter to attach console logging listeners to a Host running in an app
+ * @hidden
+ */
+import { Emitter } from "@adobe/uix-core";
+import type { HostEvents } from "./host.js";
+import type { Metrics } from "@adobe/exc-app/metrics";
+import MetricsApi from "@adobe/exc-app/metrics";
+
+type EventPayload = {
+  event: string | string[];
```

Comment:
mb in case the event is a string better convert it to string[] when we receive it and next have a strict type and always operate with it like with array, without additional checks on each usage